api/controllers/auth-controller.js

```diff
@@ -61,15 +61,28 @@ const signIn = async (req, res) => {
 const validateUser = async (req, res) => {
     try {
         const token = req.headers['access-token'];
-        if(!token) res.status(404).json({message: 'Token not found'});
+        if (!token) res.status(404).json({ message: 'Token not found' });
 
         const data = jwt.verify(token, process.env.JWT_KEY);
 
         const user = await User.findById(data.id).select("-password");
         res.status(200).json(user);
-    } catch(err) {
+    } catch (err) {
+        res.status(500).json(err);
+    }
+}
+
+const socialController = async (req, res) => {
```

Comment:
Change name socialController to social or something another

Reply:
Would you like to name it just 'social'?

Reply:
Yes, that's right